client/src/sagas/types.js

```diff
@@ -1,4 +1,21 @@
-export const LOGIN = 'LOGIN';
-export const LOGIN_ASYNC = 'LOGIN_ASYNC';
-export const REGISTER = 'REGISTER';
-export const REGISTER_ASYNC = 'REGISTER_ASYNC';
+// "reducer" actions should never be used outside of the reducers and the sagas pushing to the reducers
+export const auth = {
+  reducer: {
+    LOGIN: 'LOGIN_REDUCER',
+    LOGOUT: 'LOGOUT_REDUCER',
+    REGISTER: 'REGISTER_REDUCER',
+    LOGIN_FAILED: 'LOGIN_FAILED_REDUCER',
+    REGISTER_FAILED: 'REGISTER_FAILED_REDUCER',
+  },
+  LOGIN: 'LOGIN',
+  LOGOUT: 'LOGOUT',
+  REGISTER: 'REGISTER',
+};
```

Comment:
:oof: I don't _love_ this organization. The intent is to separate the action types that get directly handled by the reducers (and never anywhere else) from the action types that can be trigged by components. Need to think about something more elegant perhaps... or maybe @amichael497 has some insight?